packages/moonstone/TooltipDecorator/TooltipDecorator.js

```diff
@@ -366,7 +370,7 @@ const TooltipDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			forward('onKeyDown'),
 			forProp('disabled', false),
 			() => {
-				this.setTooltipLayoutJob.start();
+				this.startTooltipLayoutJob();
```

Comment:
Doesn't this change when this method fires? Instead of starting immediately it will start a job that only starts after the delay? Maybe I'm misunderstanding how these changes are structured.

Reply:
Previously, the job would not reflect a change in the `tooltipUpdateDelay` value after construction because the value was bound at construction. Now it will use the value for each invocation so the current value of the timeout is used.

Reply:
If that's the only difference, that's a nice change! Thanks!